packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/AdvancedTxDetails/IODetails/AnalyzeInExplorerBanner.tsx

```diff
@@ -25,19 +27,18 @@ const Description = styled.span`
 
 interface AnalyzeInExplorerBannerProps {
     txid: string;
+    symbol: NetworkSymbol;
 }
 
-export const AnalyzeInExplorerBanner = ({ txid }: AnalyzeInExplorerBannerProps) => {
-    const { selectedAccount } = useSelector(state => state.wallet);
-    const { network } = selectedAccount;
-    const explorerUrl = network?.explorer.tx;
+export const AnalyzeInExplorerBanner = ({ txid, symbol }: AnalyzeInExplorerBannerProps) => {
+    const explorer = useSelector(state => selectBlockchainExplorerBySymbol(state, symbol));
 
     return (
         <NotificationCard
             variant="info"
             icon="CUBE"
             button={{
-                href: `${explorerUrl}${txid}`,
+                href: `${explorer?.address}${txid}`,
```

Comment:
Should be `explorer.tx`.

![Screenshot 2024-07-16 at 14 45 31](https://github.com/user-attachments/assets/64f9ac0f-f042-4049-b8d1-e0360a11d646)